factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/validation/FactValidationError.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2017-2020 factcast.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.factcast.store.pgsql.validation;
+
+import lombok.Value;
+
+@Value
+public class FactValidationError {
+    public FactValidationError(String string) {
+        this("error", string);
+    }
+
+    public FactValidationError(String level, String string) {
+        this.level = level;
+        this.message = string;
+    }
+
+    String level;
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4877539)